### DISCUSSION TITLE
erratic wake cron

### DISCUSSION THREAD

**Discussion by kraizelburg** [seriousm4x/UpSnap#513] [Help wanted]
Hi, I have upsnap installed via docker in my rpi4 to wake my main server everyday with cron at 8am but since update to version 4.x.x this wake up call are very erratic, usually I does not wake the device however in the docker logs I can see the wake request. Then If i manually click on the wake button it works all the time and also if I save a new time in the cron, but again it does not work next day.
Any ideas? I never had such problem with version 3, I just upgraded a couple of days ago.

**Comment by seriousm4x** [comment]
#506
